server.ts

```diff
@@ -103,6 +106,7 @@ const adminPartnersMentors = new AdminPartnersMentors();
 const adminPartnersMentorStats = new AdminPartnersMentorStats();
 const adminPartnersProjects = new AdminPartnersProjects();
 const adminPartnersStudents = new AdminPartnersStudents();
+const adminPartnersCentersWithStudents = new AdminPartnersOrganizationCentersWithStudents();
```

Comment:
We can remove the `withStudents` and `withOneStudent` parts everywhere in order to not overcomplicate the names. Then we can also remove `HAVING COUNT(*) >= 1` in the query, use `getDashboardOrganizationCentres` and `getDashboardOrganizationCentresByPartnerId` as the name of the functions that are returning the list of centres, and return only the centres that have at least 1 student.

Reply:
done now :)